sag_et.par

```diff
@@ -4,9 +4,13 @@
 ! s = list of subdivisions for above division using non-native fifth
 !
 n 5 3
+|//|
 n 7 4
+|//|

```

Comment:
It is necessary to provide placeholders here so that 5 and 7 do not get merged into 9. 5 and 7 work as expected in Scala.

In the previous version of the file, when no symbols were provided, an "n" or "d" line was treated as sharing the next symbol set with all other "n" and "d" lines between it and that symbol set. However, this was only leveraged in one spot: here at the very beginning, even though it could have been used in many different spots (e.g. 31 and 38 shared symbol sets, and did not differ on what subset notations used their fifths (none did for either one) which is the only reason I can think of why EDOs sharing a symbol set would still need to be broken up). 

So it is now no longer the case that this ability is leveraged anywhere. I think that's fine. I think this is simpler, and more consistent.